conf/defaults.ini

```diff
@@ -1133,3 +1133,34 @@ default_baselayer_config =
 
 # Enable or disable loading other base map layers
 enable_custom_baselayers = true
+
+#################################### Dashboard previews #####################################
+[dashboard_previews.images]
+# Width of each image. Default is 320.
+width =
+
+# Height of each image. Default is 240.
+height =
+
+# Device scale factor passed down to the Image Renderer plugin.
+# The plugin will render a scaled up image and then scale it down to the desired width/height.
+# Default is -5.
+device_scale_factor =
+
+[dashboard_previews.crawler]
+# Number of dashboards rendered in parallel. Default is 6.
+thread_count =
```

Comment:
Can we wait on these settings -- I don't want to add "preview" specifc settings while we are still sorting out how to best do search/discovery/crawler etc

Reply:
👍 will keep this PR in draft state just for reference